### PR TITLE
Added new function reportStatus in utils.js

### PR DESCRIPTION
**Pull request by vcollantes**

### CODE REVIEW

File: src/webview-bridge-cjs.js.flow [hunk 1]
```diff
@@ -121,3 +121,9 @@ declare export function updateNavigationBar({|
 |}): Promise<void>;
 
 declare export function isABTestingAvailable(key: string): Promise<boolean>;
+
+declare export function reportStatus({|
+    status: 'CRITICAL' | 'BAD' | 'GOOD',
+    reason: string,
+    feature: string,
```
Comment (SergioMorchon) [inline src/webview-bridge-cjs.js.flow]:
```typescript
feature: 'ACCOUNT';
```

File: src/utils.ts [hunk 2]
```diff
@@ -95,3 +95,17 @@ export const isABTestingAvailable = (key: string): Promise<boolean> => {
         return Promise.resolve(isRemoteConfigAvailable(key));
     }
 };
+
+export const reportStatus = ({
+    feature,
+    status,
+    reason,
+}: {
+    feature: string;
```
Comment (SergioMorchon) [inline src/utils.ts]:
```typescript
feature: 'ACCOUNT';
```

File: src/post-message.ts [hunk 3]
```diff
@@ -86,6 +86,15 @@ type MessageTypeToResponseMap = {
         type: 'GET_REMOTE_CONFIG';
         payload: {result: {[s: string]: string}};
     };
+    STATUS_REPORT: {
+        id: string;
+        type: 'STATUS_REPORT';
+        payload: {
+            status: 'CRITICAL' | 'BAD' | 'GOOD';
+            reason: string;
+            feature: string;
```
[the remark this answers is not in the document]
Comment (vcollantes) [reply src/post-message.ts]: Ok, for me. The only inconvenient I have with this is to change this repo to add another literal. WDYT @pladaria ???

[the remark this answers is not in the document]
Comment (pladaria) [reply src/post-message.ts]: +1 to Sergio's suggestion

[the remark this answers is not in the document]
Comment (vcollantes) [reply src/post-message.ts]: ok, thanks!

File: README.md [hunk 4]
```diff
@@ -390,6 +390,24 @@ setScreenName('Topup Flow').then(() => {
 });
 ```
 
+### reportStatus
+
+Sends a postmessage to the app with the status,
```
[the remark this answers is not in the document]
Comment (vcollantes) [reply README.md]: ok!

File: README.md [hunk 5]
```diff
@@ -390,6 +390,24 @@ setScreenName('Topup Flow').then(() => {
...
+
+-   Available for app versions 11.2 and higher
+
+```typescript
+reportStatus: ({status: 'CRITICAL' | 'GOOD' | 'BAD', reason: string, feature: string}) => Promise<void>;
```
[the remark this answers is not in the document]
Comment (vcollantes) [reply README.md]: ok! Thanks :)

File: README.md [hunk 6]
```diff
@@ -390,6 +390,24 @@ setScreenName('Topup Flow').then(() => {
...
+```
+
+#### Example
+
+```javascript
+import {reportStatus} from '@tef-novum/webview-bridge';
+
+reportStatus({status: 'GOOD', reason: 'whatever', feature: 'ACCOUNT'});
```
[the remark this answers is not in the document]
Comment (vcollantes) [reply README.md]: ok!

File: src/post-message.ts [hunk 7]
```diff
@@ -86,6 +86,15 @@ type MessageTypeToResponseMap = {
         type: 'GET_REMOTE_CONFIG';
         payload: {result: {[s: string]: string}};
     };
+    STATUS_REPORT: {
+        id: string;
+        type: 'STATUS_REPORT';
+        payload: {
```
[the remark this answers is not in the document]
Comment (vcollantes) [reply src/post-message.ts]: Yes, right @pladaria. I thought this structure was to send, but as we talked in private this is just the response as the name says.

**Pull request merged**